### ISSUE TITLE
impossible to install with last Ubuntu 20.04

### ISSUE THREAD

**Issue by franky5912** [sanette/intuos4-oled#2]
Hi @sanette 
I was using your intuos4-oled program with my previous Ubuntu version, the 18.04. It was perfectly working, thanks a lot, great job !

Since few days, I switched to Ubuntu 20.04 and unfortunately, I'm no longer able to install it. Not particular issue during the installation. No error while installing at and pytho-pil, no error while running the install.sh.

However, when I try running the script intuos4oled.py, I always get the message that the file do not exists while the file is there !

> Installation completed. You may now plug the Intuos in.

> franky@maison:/tmp/intuos4-oled-master$ sudo ./intuos4oled.py init

> sudo:` exécution de ./intuos4oled.py impossible: Aucun fichier ou dossier de ce type`

Would you have any idea about this issue ?

**Comment by sanette** [comment]
Hi @franky5912 , nice to hear that the program was useful to you.
Did you install python?
`/usr/bin/python`

**Comment by sanette** [comment]
maybe you need to select which python version you want to use (2 or 3).
Try:
`sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1`

https://linuxconfig.org/ubuntu-20-04-python-version-switch-manager

**Comment by franky5912** [comment]
Hi @sanette, yes I had installed python and before you answered I found the possibility for switching version. With Python 2 I've been able to change images on buttons, nice !!. 
However, I faced other issues:
- it is not possible to add text instead of an image:
```
franky@maison:~/Installations/intuos4-oled-master$ intuos4oled.py set -t "ANNULER" -b 0`
Using Wacom Intuos4 (8x13) (PTK-840)
Active led = 1
Loading datafile /home/franky/.intuos
Sending "ANNULER" to button 0
Traceback (most recent call last):
  File "/usr/local/bin/intuos4oled.py", line 489, in <module>
    span = args.span, font = args.font, size = None)
  File "/usr/local/bin/intuos4oled.py", line 407, in send_text
    text_to_img(text, filename, font, size, span)
  File "/usr/local/bin/intuos4oled.py", line 390, in text_to_img
    ret = subprocess.call(args1 + resize + args2)
  File "/usr/lib/python2.7/subprocess.py", line 172, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```
- when I push on the central ring button for switching from config 1 to 2, 3 or 4, I loose the images when coming back on the config 1. I can get back all my images only by setting again one of them or trying to set (vainly) a text. So I understand that calling the intuos4oled.py is sufficient to reload the configuration of the buttons.
- init also is not functionning : 
```
franky@maison:~/Installations/intuos4-oled-master$ intuos4oled.py set -t "ANNULER" -b 0
Using Wacom Intuos4 (8x13) (PTK-840)`
Active led = 1
Loading datafile /home/franky/.intuos
Sending "ANNULER" to button 0
Traceback (most recent call last):
File "/usr/local/bin/intuos4oled.py", line 489, in <module>
    span = args.span, font = args.font, size = None)
 File "/usr/local/bin/intuos4oled.py", line 407, in send_text
    text_to_img(text, filename, font, size, span)
 File "/usr/local/bin/intuos4oled.py", line 390, in text_to_img
    ret = subprocess.call(args1 + resize + args2)
 File "/usr/lib/python2.7/subprocess.py", line 172, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

As I'm not a python programmer I can not enter into your program to try investigate on pissible other conflict :-(

**Comment by sanette** [comment]
for the first point: the program is using `convert` from imagemagick, 
maybe you need to install the compatibiliy package `graphicsmagick-imagemagick-compat`

for the second point (init), can you paste the correct error message?

**Comment by franky5912** [comment]
Thanks @sanette !!
I tried `graphicsmagick-imagemagick-compat` and was finally able to show a text. However the size was not adjusted (my "ANNULER" was shown as "NNULER" on the tablet).
Then I decided to install the package `imagemagick` and now it's running fine (texts and images). In fact I thought that `imagemagick` was already installed but not: the package installed originally for images manipulations was `graphicsmagick`.
Once again, thanks a lot for your support and this nice program ;-)
